src/redux/actions/sync.js

```diff
@@ -31,6 +31,36 @@ export function doSetSync(oldHash, newHash, data) {
   };
 }
 
+export function doSetDefaultAccount() {
+  return dispatch => {
+    dispatch({
+      type: ACTIONS.SET_DEFAULT_ACCOUNT,
```

Comment:
Nothing is listening for this? Do we need it?

Reply:
Just added this in case we may have to use it in the future.